### ISSUE TITLE
[GSSoc'21]Font sizes are not perfect in phone view 

### ISSUE THREAD

**Issue by arpit456jain** [swapnilsparsh/Rescue#186]
**Describe the bug**
----
The font size is hard-coded in every tag like h1 and p , it is ok for desktop version but in small screen like phone the text is not clearly visible It is looking like this.
![screencapture-127-0-0-1-8000-women-rights-2021-03-21-15_43_27](https://user-images.githubusercontent.com/55352601/111901587-23146f00-8a5f-11eb-8ed0-11009d670e51.png)
 

**To Reproduce**
----
- Steps to reproduce the behavior:

**Expected behavior**
-----
The font size should automatically increase in small screen. 

**Possible solution**
----
- Describe the solution you thought of.
I had already solved this issue by writing some media query and remove all inline CSS and make it internal CSS for easy editing . Now it is looking like this 

https://user-images.githubusercontent.com/55352601/111902662-8785fd00-8a64-11eb-9cb4-e78be8b83e7d.mp4

I am gsoc'21  participant  please assign me this issue and allow me to create pull request .
I had already done this on my local machine all i have to do is create a pull request if assign to me. 

- Add any other context about the problem here.You can also mention your system environment in which you are running it.

**Comment by arpit456jain** [comment]
Please assign me this issue i will also modify the font size of other two pages within an hour .

**Comment by sakshikhachane** [comment]
@arpit456jain Actually there is no such font size problem. It is looking as it is expected to be.

**Comment by arpit456jain** [comment]
Actually Its was very small in Phone for me when i hosted it and opened it in my phone every thing is very small ,
no problem if you think it's fine 🙂 you can close this issue.

**Comment by sakshikhachane** [comment]
@arpit456jain Go ahead with the issue. The font sizes are `hard-coded` and ideally, it shouldn't be. Happy learning:)

**Comment by arpit456jain** [comment]
Ok mam 
Mam can you please add the label of (level)

**Comment by arpit456jain** [comment]
@sakshikhachane  I had almost solve this issue **and improve the design of navbar if you notice currently in phone view when we click on toggle button it leave some space on both sides, I had solved it**
I had fixes the 
home.html
common.html
currently working on the files which extends common.html
I have a request **Please don't Merge any PR until I create a PR otherwise I have to do all changes again Which  will be very difficult for me**
just give me an hour Please

**Comment by arpit456jain** [comment]
I had replace all inline CSS **(which are hard-coded)** to internal CSS for easy editing and media queries  Please don't merge any PR

**Comment by arpit456jain** [comment]
@jatinjain001 sir please review the PR #195  before any other PR got merge otherwise i have to do changes again if any conflicts occur

**Comment by arpit456jain** [comment]
@sakshikhachane mam please review my PR #195

**Comment by tanishq-arya** [comment]
Please assign this to me under gssoc

**Comment by sakshikhachane** [comment]
Go ahead @tanishq-arya.

**Comment by tanishq-arya** [comment]
I'm sorry for the delay. I was not well. Will raise PR by end of the week.
